packages/expect/src/utils.ts

```diff
@@ -133,6 +133,13 @@ export const getObjectSubset = (object: any, subset: any): any => {
   return object;
 };
 
+// NOTE: Should be removed after dropping node@6.x support,
+// and we should use (Object.entries)
+export const getObjectEntries: typeof Object.entries = (
+  object: any,
+): Array<[string, any]> =>
+  object == null ? [] : Object.keys(object).map(key => [key, object[key]]);
```

Comment:
Instead of `Object.keys` do we need to call `keys` in `jasmineUtils.ts` to include enumerable symbols (and maybe filter out `Symbol.iterator` in rare case it might be iterable in user-defined object) so that the comparison of properties is as consistent with ordinary `equals` call? I am asking, not telling.

This illustrates a limitation we have recently noticed: it is not (yet) possible for custom tester to know additional arguments for property comparison which distinguish `toEqual` from `toStrictEqual` matcher. Ignore that problem.